src/core/mod.rs

```diff
@@ -1,6 +1,6 @@
-mod cigar;
-mod counter;
-mod diger;
-mod matter;
-mod util;
-mod verfer;
+pub mod cigar;
+pub mod counter;
+pub mod diger;
+pub mod matter;
```

Comment:
Does Matter want to be public? I wouldn't expect people to be initializing it as it's a base for the others?

Reply:
Possibly not, but in the Rust syntax we'd probably need a type because using a trait you initialize like this:
```rust
let verfer = <Matter as Verfer>::new_with_code_and_raw(/* ... */);
```

.. we may be able to alias all these? I can look into this in another issue.